src/popup/router/routes.ts

```diff
@@ -275,8 +275,25 @@ export const routes: WalletAppRouteConfig[] = [
         },
       },
       {
-        path: 'coins/:id',
+        /**
+         * When a route is defined with a parameter
+         * and user leaves this route, vue-router will
+         * throw a "missing required param" error
+         * even though the parameter was set for the route
+         * Making the parameter optional & checking for it
+         * in the beforeEnter hook fixes the issue
+         *
+         * @see https://github.com/vuejs/router/issues/845
+         */
```

Comment:
I suggest to use full available width of 100 chars for the comment because right now it looks little bit too dense in my opinion.